Contest_API.md

```diff
@@ -2183,17 +2184,34 @@ state, but never an earlier state.
 ##### Reconnection
 
 If a client loses connection or needs to reconnect after a brief
-disconnect (e.g. client restart), it can use the 'time' parameter to
-specify the last event it received:
+disconnect (e.g. client restart), it can use the 'since_token' parameter to
+specify the last notification token it received:
 
-`/event-feed?time=xx`
+`/event-feed?since_token=xx`
 
 If specified, the server will attempt to start sending events around the
-given time to reduce the volume of events and required reconciliation.
-If the time passed is too large or the server does not support this
-parameter, all objects will be sent. There is no guarantee that all
-updates (e.g. a team name correction, which is not time-based) that
-occurred during the time the client was disconnected will be reflected.
+given token to reduce the volume of events and required reconciliation.
+If the token is invalid, the time passed is too large (typically no more
+than 15 minutes), or the server does not support this parameter, the
+request will fail with a 400 error.
+
+There is no guarantee that all updates (e.g. a team name correction,
+which is not time-based) that occurred during the time the client was
+disconnected will be reflected. The server may also resend some
+notifications to ensure that they were received.
```

Comment:
So changes in static data (like a team name change) are tricky, but I think it should be guaranteed at least that the server will provide all updates to live data required to get into a consistent state again with reality.

The reason is that also the "start sending events _around_" is a bit vague. Currently it doesn't seem guaranteed that the server doesn't just skip a few events after `since_token`.

Reply:
Yeah, I think we need a *little* bit stronger guarantees.

How about something like: if `since_token` is used the client will either get a 400 *or* it is guaranteed that it will get at least all changes since the token (but it *could* also get a lot more).

Reply:
Please review again. I removed the word 'around' and used Fredrik's text. I felt it needed a reason why it might receive more events so included that as well.